tests/integration-tests/conftest.py

```diff
@@ -271,29 +272,41 @@ def cfn_stacks_factory():
     factory.delete_all_stacks()
 
 
+_AVAILABILITY_ZONE_OVERRIDES = {
+    # c5.xlarge is not supported in us-east-1e
+    # FSx Lustre file system creation is currently not supported for us-east-1e
+    "us-east-1": ["us-east-1a", "us-east-1b", "us-east-1c", "us-east-1d", "us-east-1f"],
```

Comment:
We could add a link to this on our documentation, as a see what az's fsx is in.

Reply:
according to https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html: 

> An Availability Zone is represented by a region code followed by a letter identifier; for example, us-east-1a. To ensure that resources are distributed across the Availability Zones for a region, we independently map Availability Zones to names for each AWS account. For example, the Availability Zone us-east-1a for your AWS account might not be the same location as us-east-1a for another AWS account.

which means probably also having it hardcoded like this is not the best option. Maybe an account based config file might be a better way. Shall we leave it like this for now?

Reply:
added a FIXME comment